setup.py

```diff
@@ -41,17 +41,17 @@ def open_local(paths, mode="r", encoding="utf8"):
         "Development Status :: 3 - Alpha",
         "Environment :: Web Environment",
         "License :: OSI Approved :: MIT License",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
     ],
 }
 requirements = [
-    "httpx==0.18.*",
-    "websockets>=9.0",
+    "httpx",
+    "websockets",
 ]
 
-tests_require = ["pytest", "sanic==21.3", "pytest-asyncio"]
+tests_require = ["pytest", "sanic", "pytest-asyncio"]
```

Comment:
Let's make is min version Sanic 21.3. We will not support earlier versions here since those versions have this lib integrated.